best_practices/forms.rst

```diff
@@ -121,7 +122,7 @@ some developers configure form buttons in the controller::
         {
             $post = new Post();
             $form = $this->createForm(PostType::class, $post);
-            $form->add('submit', 'submit', array(
+            $form->add('submit', SubmitType::class, array(
```

Comment:
I think we need to show the 5.5 syntax, because it's almost unusable if you can't use this syntax. But the idea is to also show the 5.3 syntax with a note about this in some cases early on in chapters. Basically, I want people to know that they need 5.5 to use the new syntax, but that they can use the old aliases until they actually need to upgrade to Symfony 3 (which requires 5.5). It's tricky.

And I think the `use` statement is here - for `SubmitType` and `PostType`?